lib/ex_doc/retriever.ex

```diff
@@ -314,4 +365,19 @@ defmodule ExDoc.Retriever do
   defp source_link(%{url_pattern: url_pattern, relative_path: path}, line) do
     url_pattern.(path, line)
   end
+
+  defp normalize_group(group) do
+    case group do
+      %{title: title, description: description}
+      when is_binary(title) and (is_binary(description) or is_nil(description)) ->
+        group
+
+      kw when is_list(kw) ->
+        true = Keyword.keyword?(kw)
+        %{title: Keyword.fetch!(kw, :title), description: kw[:description]}
+
+      title when is_binary(title) when is_atom(title) ->
+        %{title: title, description: nil}
```

Comment:
We should probably normalize the title here?

Reply:
You mean accept atoms and convert to binary if needed?

Reply:
Right now it is already accepting atoms so normalizing could be nice? :)